src/main/java/org/candlepin/model/DeletedConsumerCurator.java

```diff
@@ -100,11 +106,21 @@ public int createDeletedConsumers(Collection<String> consumerIds) {
     }
 
     @SuppressWarnings("unchecked")
-    public CandlepinQuery<DeletedConsumer> findByDate(Date date) {
```

Comment:
Why was this method updated, it doesn't look like it was affected by the `listAll` and `listAllByIds` changes?

Reply:
It looks like this was updated for the changes in DeletedConsumerResource

Reply:
This method is used in the resource where it uses listAll and this method as a result of a condition. They must return the same object.